.github/workflows/cherry-pick-next-to-master.yml

```diff
@@ -23,6 +25,7 @@ jobs:
           branch: master
           body: 'Cherry-pick of #{old_pull_request_id}'
           cherry-pick-branch: ${{ format('cherry-pick-{0}', github.event.number) }}
+          title: '{old_title} (@${{ github.event.pull_request.user.login }})'
```

Comment:
https://github.com/LukasTy/test-github-action/pulls
Include `(@<cherry_pick_pr_author_login>)` in the [PR title](https://github.com/LukasTy/test-github-action/pull/14).
Any other options did not seem to work, because even having both `committer` and `author` fields set in the action, would still yield a PR [created by the bot](https://github.com/LukasTy/test-github-action/pull/11). 🙈